storage/queued_country.hpp

```diff
@@ -6,6 +6,9 @@
 namespace storage
 {
 /// Country queued for downloading.
+/// @TODO(bykoianko) This class assume that a map may consist of one or two mwm files.
+/// But single mwm files are used now. So this class should be redisigned to support
```

Comment:
> But IMHO it should be done by adding ASSERTions that CarRouting is never requested during QueuedCountry usage.

Yes. I agree. I'll try to add that. It could lead to removing some unnecessary test. Then (I hope before the release) I'll refactor queued_country.